src/NUnitEngine/nunit.engine.tests/Services/TestRunnerFactoryTests/TestCases/NetStandardTestCases.cs

```diff
@@ -79,17 +79,10 @@ public static IEnumerable<TestCaseData> TestCases
                     new RunnerResult
                     {
                         TestRunner = typeof(AggregatingTestRunner),
-                        SubRunners = new List<RunnerResult>
+                        SubRunners = new []
```

Comment:
`new []` is an inconsistent style compared to existing and added lines. (Several occurrences)

Reply:
Yeah, it's used more consistently in the cases and results on the .NET Framework side. I like [] better but I probably shouldn't have changed it arbitrarily. That said, I strongly suspect we will be reworking all these things - and they may have already been changed in the dev-4.0 branch because I made lots of changes there.